build.sbt

```diff
@@ -114,14 +114,14 @@ lazy val `clio-client` = project
   .settings(commonTestDockerSettings)
 
 /**
-  * Integration tests for the clio-server (and eventually the clio-client).
+  * Integration tests for the clio-server and the clio-client.
   */
 lazy val `clio-integration-test` = project
-  .enablePlugins(DockerPlugin)
-  .settings(commonDockerSettings)
-  .settings(dockerfile in docker := Docker.integrationTestDockerFile.value)
+  .dependsOn(`clio-client`, `clio-dataaccess-model`)
```

Comment:
Not going to change this now because to do it properly we'd need to rename the directory too, and I don't want to cause more merge conflicts for people working in the elasticsearch code.